public/cli/lib/utility/checkDev.ts

```diff
@@ -0,0 +1,10 @@
+export default async function checkDev(): Promise<boolean> {
+  // Importing isDev causes an error running tests as we don't run in an
+  // electron environment. Ava sets the NODE_ENV to 'test'.
+  if (process.env.NODE_ENV === 'test') {
+    return false;
+  } else {
+    const isDev = await import('electron-is-dev');
```

Comment:
This would re-import if the function was called again. Not a problem though for the way it is used at the moment.

Reply:
I took the liberty of fixing this myself since it was the last thing before merging.